CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 This is a history of changes to clara-rules.
 
+### 0.17.0
```

Comment:
Minor thing I missed before, but we should probably label this "0.17.0-SNAPSHOT" to avoid giving the impression that we've released 0.17.0.  It will then be changed when we do a release of course.